pyproject.toml

```diff
@@ -79,8 +79,16 @@ fastapi-utils = { version = "~0.2.1", optional = true}
 
 # optional dependencies for stack recipes
 
+# Optional dependencies for project templates
+copier = ">=7.0.0"
```

Comment:
I think these are also all non-optional right now? See the `optional=true` used above

Reply:
Yes, correct. I made them all optional.